airgun/entities/discoveryrule.py

```diff
@@ -89,6 +92,48 @@ def disable(self, entity_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def discovered_hosts(self, entity_name):
```

Comment:
I think that is more respect our approach
```
    def read_discovered_hosts(self, entity_name):
        view = self.navigate_to(self, 'Discovered Hosts')
        view.flash.assert_no_error()
        view.flash.dismiss()
        return view.read()
```
```
navigation:
view = DiscoveredHostsView
pre_requisite:
        view = self.navigate_to(self, 'All')
step:
        parent_view.table.row(name=entity_name)[ACTION_COLUMN].widget.fill(
            'Discovered Hosts')
        wait_for(
            lambda: view.is_displayed is True,
            timeout=60,
            delay=1,
            logger=discovered_hosts_view.logger
        )
```

Reply:
+1